README.md

```diff
@@ -182,6 +182,8 @@ more information about the governance of the Node.js project, see
 **Ali Ijaz Sheikh** &lt;ofrobots@google.com&gt;
 * [rvagg](https://github.com/rvagg) -
 **Rod Vagg** &lt;rod@vagg.org&gt;
+* [thefourtheye](https://github.com/thefourtheye) -
```

Comment:
bleh, I knew I'd get this wrong, can never seem to get it right the first go! updated